{
  Debug.logInfo("Running testDynamicAccessFailure()",module);
  assertFalse("User already has the auto-granted permission",security.hasPermission("system","test:autogranted",null));
  assertTrue("User was not granted dynamic access as expected",security.hasPermission("system","test:groovy1:1000",null));
  assertTrue("User was not auto-granted expected permission",security.hasPermission("system","test:autogranted",null));
}
